dask/dataframe/core.py

```diff
@@ -540,9 +540,79 @@ def sample(self, frac, replace=False, random_state=None):
         return self._constructor(merge(self.dask, dsk), name,
                                  self._pd, self.divisions)
 
-    @derived_from(pd.DataFrame)
     def to_hdf(self, path_or_buf, key, mode='a', append=False, complevel=0,
                complib=None, fletcher32=False, get=None, **kwargs):
+        """ Export frame to hdf file(s)
+
+        Export dataframe to one or multiple hdf5 files or nodes.
+
+        Exporting dataframe to multiple targets enables improved parallelism
+        and is expected to provide better performance for longer computations.
+
+        Parameters
+        ----------
+        path_or_buf: HDFStore object or string
+            Destination file(s). If string, can contin a single wildcard
+            to save each partition to a different file. Only one widlcard
+            is allowed in both path and key,
+        key: string
+            A node / group identifier, can contain a single wildcard
+            to save each partition to a different hdf node in a single
+            file. This multiple better parallelism compared to saving
+            entire dataframe in a single node.
+        mode: optional, {'a', 'w', 'r+'}, default 'a'
+
```

Comment:
That's how pandas have it, so i did the same.
i have no clue if it's really needed, to be honest, and would go either way if someone has any opinion/preference.